bots/tests-scan

```diff
@@ -49,6 +49,7 @@ REDHAT_VERIFY = {
     "verify/rhel-x": [ 'master' ],
     "verify/rhel-atomic": [ 'master' ],
     'selenium/explorer': [ 'master' ],
+    'selenium/edge': [ 'master' ]
```

Comment:
Please leave the list empty for now, as the test does not really exist yet. You can add the "master" in the real PR that adds it.

Reply:
ahhh ok